### ISSUE TITLE
Per-baseline noise ratio plots

### ISSUE THREAD

**Issue by o-smirnov** [atemkeng/Windowing-Functions#11]
I'm confused by the per-baseline noise plots. X axis says wavelengths, goes to 120 or so. At 1.4 GHz lambda is 21cm, so that's a max baseline of 24 metres?? Can't be right.

**Comment by atemkeng** [comment]
Definition 1.1. (Time direction spacing)  defined $d_t$.  In fact, X axis is a sort average , so if the longest baseline takes ~ $d_t$=24 metres to move during the integration time t.  For N timeslots, the average values of $d_t$ is  ~24 metres. In fact X axis is not the baseline length but the average distance that each baseline takes to sweep the uv domain for N timeslots. I will reconsider the issue again!

**Comment by o-smirnov** [comment]
Ah ok makes sense!

**Comment by o-smirnov** [comment]
Might be more natural to express it as the u (east West) component of the
baseline, since that is perfectly proportional to this quantity.

**Comment by atemkeng** [comment]
Ok, I will do that, because it is confusing and the simple case I mentioned on this issues is only the case of a E-W baseline tracking a source at the Zenith......

**Comment by o-smirnov** [comment]
Actually I'm talking bollocks, u is not the E-W component, u changes and is E-W on the sky not on Earth... so you need to work out the E-W component based on ANTENNA coordinates.

While we're at it, I'm actually confused about the plots. Each point is a single baseline? But then, how do you compute the noise on that baseline? You must have a number of visibilities to take the rms over, correct?

**Comment by atemkeng** [comment]
Yes, but I compute the noise ratio. Look Eq.44, by doing the ratio, the noise per visibility is simplify  $\sigma_s^2$. That is exactly what I did in my code.

**Comment by o-smirnov** [comment]
Ahh, you mean this is the analytic prediction?

**Comment by atemkeng** [comment]
Yes the plots are for the analytic then I compared the overall (a sort of average \sqrt{n1 +n2 +n3 + ......}) to the simulation one.
